fuzzers/libfuzzer/fuzzer.py

```diff
@@ -53,6 +53,13 @@ def run_fuzzer(input_corpus, output_corpus, target_binary, extra_flags=None):
     os.makedirs(crashes_dir)
     os.makedirs(output_corpus)
 
+    # Enable symbolization if needed.
+    for flag in extra_flags:
+        if flag.startswith('-focus_function'):
+            os.environ['ASAN_OPTIONS'] += ':symbolize=1'
```

Comment:
Maybe leave a comment that if `symbolize=0:..:symbolize=1` then only `symbolize=1` is respected.